### PR TITLE
update the entrepreneur flow with stripe connect

### LINKED ISSUE #114: Add stripe connection in the business registration flow

### PR DESCRIPTION
**Pull request by petarblazevski**
Closes #114

### CODE REVIEW

File: src/components/Loader.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<style scoped>
+.spinner {
+  width: 50px;
+  height: 30px;
+  margin: 0 auto;
+  font-size: 10px;
+  text-align: center;
+}
+
+.spinner > div {
```
Comment (IAfanasovMob) [inline src/components/Loader.vue:25]: it,s scss, mate 🙃 
```
& > div {
}
```

File: src/components/entrepreneur-flow/EntrepreneurFlowStripeConnect.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  components: {
+    Loader,
+  },
+  data() {
+    return {
+      isLoading: false,
+    }
+  },
+  methods: {
+    async getConnectUrl() {
```
Comment (IAfanasovMob) [inline src/components/entrepreneur-flow/EntrepreneurFlowStripeConnect.vue:41]: `redirectToConnectUrl`?

File: src/views/Entrepreneur.vue [hunk 3]
```diff
@@ -18,26 +22,23 @@
...
     EntrepreneurFlowPersonName,
     EntrepreneurFlowBusinessInfo,
-    EntrepreneurFlowSetupPayment,
-    EntrepreneurFlowFinish,
+    EntrepreneurFlowStripeConnect,
   },
   data: () => {
     return {
-      step: 1,
+      step: 2,
```
Comment (IAfanasovMob) [inline src/views/Entrepreneur.vue:37]: `0`?

File: src/components/Loader.vue (hunk 1 above)
Comment (petarblazevski) [reply src/components/Loader.vue:25]: copy-paste generation 😆

Comment (IAfanasovMob) [reply src/components/Loader.vue:25]: feel free to leave it as is 🙃 
I find it useful to add a reference to the source of copy. could help somewhere in the future
not a blocker as well 🙃

**Pull request merged**